File: src/main/FizzBuzz.js
for(let x = 1; x <=100; x++)
{
    let output = "";

    if(x % 3 == 0 && x % 5 == 0)
        console.log(output += "fizzbuzz");
    else if(x % 3 == 0)
        console.log(output += "fizz");
    else if(x % 5 == 0)
        console.log(output += "buzz");        
    else if(x % 7 == 0)
        console.log(output += "bazz");
    else
        console.log(x)
}

File: src/main/FizzBuzz.java
package main;
public class FizzBuzz{
public static void main (String [] args){
//Print 1 to N numbers but print fizz if the number is diclsvisible by 3 and buzz if it's divisable by 5

//Declaring variables
    int N = 100; 
    String fizz = "fizz", buzz = "buzz", fizzbuzz = "fizzbuzz";

for(int x = 1; x <= N; x++)
{
    if(x % 3 == 0 && x % 5 == 0)
        System.out.println(fizzbuzz);
    else if(x % 3 == 0)
        System.out.println(fizz);
    else if(x % 5 == 0)
        System.out.println(buzz);
    else 
        System.out.println(x);
}

    }
}